src/hotspot/cpu/riscv/interpreterRT_riscv.cpp

```diff
@@ -227,7 +227,7 @@ class SlowSignatureHandler
 
   virtual void pass_object() {
     intptr_t* addr = single_slot_addr();
-    intptr_t value = *addr == 0 ? NULL : (intptr_t)addr;
+    intptr_t value = *addr == 0 ? nullptr : (intptr_t)addr;
```

Comment:
PS: I got compile errors when doing a native build with GCC-11.3.0:
```
474 /home/fyang/openjdk-jdk/src/hotspot/cpu/riscv/interpreterRT_riscv.cpp: In member function 'virtual void SlowSignatureHandler::pass_object()':
475 /home/fyang/openjdk-jdk/src/hotspot/cpu/riscv/interpreterRT_riscv.cpp:230:33: error: operands to '?:' have different types 'std::nullptr_t' and 'intptr_t' {aka 'long int'}
476   230 |     intptr_t value = *addr == 0 ? nullptr : (intptr_t)addr;
477       |                      ~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~
```

I think we should change the "nullptr" here into "(intptr_t)nullptr".

Reply:
Yeah, that's correct. Fixed!